internal/kafka/internal/api/dbapi/data_plane_cluster_status.go

```diff
@@ -30,10 +30,11 @@ type DataPlaneClusterStatusResizeInfo struct {
 }
 
 type DataPlaneClusterStatusCapacity struct {
-	IngressEgressThroughputPerSec string
-	Connections                   int
-	DataRetentionSize             string
-	Partitions                    int
+	IngressThroughputPerSec string
+	EgressThroughputPerSec  string
```

Comment:
Looking at the change https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/pull/663 on the FSO side, only the `.Spec` part is changed not the `.Status`